backend/src/main/java/eventlistener/service/notificaionuser/NotificationUserService.java

```diff
@@ -1,18 +1,22 @@
 package eventlistener.service.notificaionuser;
 
 import eventlistener.exception.EventNotFoundException;
-import eventlistener.model.notificationUser.NotificationUser;
-import eventlistener.model.notificationUser.NotificationUserDTO;
+import eventlistener.model.event.Event;
+import eventlistener.model.notificationuser.NotificationUser;
+import eventlistener.model.notificationuser.NotificationUserDTO;
+import eventlistener.model.notificationuser.NotificationUserEditDTO;
 import eventlistener.repo.EventRepo;
 import eventlistener.repo.NotificationUserRepo;
 import eventlistener.service.event.EventService;
+import lombok.extern.slf4j.Slf4j;
 import org.springframework.stereotype.Service;
 
 import java.util.List;
 import java.util.NoSuchElementException;
 import java.util.Optional;
 
```

Comment:
```suggestion
import ...NotificationUserMapper.*
```
then you can use all the static methods without autowiring the mapper and prefixing all the methods with notificationUserMapper